src/SkiddleBase.php

```diff
@@ -155,8 +258,8 @@ protected function makeCall($endpoint, $asArray = false)
         $args = $this->conditions;
 
         //QUIRK: API expects eventcode to be uppercase
-        if (isset($args['eventcode'])) {
-            $args['eventcode'] = strtoupper($args['eventcode']);
+        if (isset($args['eventCode'])) {
```

Comment:
Is it possible that we need to support `eventcode` too? Wondering if we should add a new conditional rather than replacing the existing one, but I don't know the extent of where/how this is used.

Reply:
I thought about that - but as this is an major version update, breaking changes are allowed

Reply:
Yeah, I'm content with breaking changes, just wasn't sure if say... another API call needed `eventcode` while the new event search needed `eventCode`. Just want to avoid breaking anything :)